### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by Abrar052**
###Microverse Portfolio Project (Mobile version)
-Updated index.html by Adding Next Two Sections
-Updated style.css by Adding Style to New Two Sections 
-Updated Json Packages
 -Updated .gitignore
 -Added Images to the New Sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,11 +40,15 @@ <h3>I am a Software Developer</h3>
         </section>
     </div>
     
-    <div class="container-2">
+    <div class="grid-container">
```
Comment (prolajumokeoni) [inline index.html:43]: - [ ] The container of the section is a section tag, please change the main wrapper of this section to a section tag, based on the requirements and for semantic purposes 👍🏿 
- [ ] Please align all content in this section to the center currently your cards are not aligned properly, see screen shot below 👍🏿  


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Template 2-Main-Mobile |  Screenshot_307

File: index.html [hunk 2]
```diff
@@ -56,17 +60,66 @@ <h2>Multi-Post Stories</h2>
...
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                
+                    <button class="project" type="button">See Project </button>
+                </div>
+                
+            </div> 
+        </div>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly duplicate this card to be six just like in the Figma design, see screenshot above  👍🏿

File: index.html [hunk 3]
```diff
@@ -56,17 +60,66 @@ <h2>Multi-Post Stories</h2>
...
+            
+    
+    
+
+    </div>
+    <div class="container-3">
```
Comment (prolajumokeoni) [inline index.html:124]: - [ ] The container of the section is a section tag, please change the main wrapper of this section to a section tag, based on the requirements and for semantic purposes 👍🏿

File: index.html [hunk 4]
```diff
@@ -56,17 +60,66 @@ <h2>Multi-Post Stories</h2>
...
+            <div class="list-3">
+                <h3 class="skills"> Skills</h3>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
```
Comment (prolajumokeoni) [inline index.html]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Template 2-Main-Mobile | Screenshot_308

- [ ] Kindly align the background image to the right, please make your design look like the Figma
- [ ] Please align languages, frameworks and skills to the center and add icons to each of them 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Abrar052, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design 



## Required Changes ♻️

- [ ] Kindly make sure that git properly ignores node modules, currently, some files from node modules are still pushed to git, this may happen [if you had commited them in previous commit to git](https://gist.github.com/pavankjadda/2bb6fbdd8786e1f57fd7bcbcc666b51d). Please check fix this 🤝

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -40,7 +40,8 @@ <h3>I am a Software Developer</h3>
         </section>
     </div>
     
-    <div class="container-2">
+    <div class="grid-container">
```
Comment (topeogunleye) [inline index.html:43]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you change this `div` tag to a `section` tag.

File: index.html [hunk 6]
```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
...
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
                 </div>
             </div> 
+        </section>
+    </div>
+    <div class="container-3">
```
Comment (topeogunleye) [inline index.html:124]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you change this`div` tag to a`section` tag for [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) purposes.

File: index.html [hunk 7]
```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
...
+        <div class="About">
+            <h1>About Me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <BUtton class="resume">Get My Resume</BUtton>
+            <img src="assets/Frame.png"  class= "write" alt="">
+            
+            
+            <section class="final">
+                <div class="list-1">
+                    <h3 class="lan">Languages</h3>
```
Comment (topeogunleye) [inline index.html:134]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you add icons for language, framework, and skills.
image

File: index.html [hunk 8]
```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
...
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                    </ul>
+                </div>
+                <div class="list-2">
+                    <h3 class="fram"> Framworks</h3>
```
Comment (topeogunleye) [inline index.html:143]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you add icons for language, framework, and skills.
image

File: index.html [hunk 9]
```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
...
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="list-3">
+                    <h3 class="skills"> Skills</h3>
```
Comment (topeogunleye) [inline index.html:153]: - [ ] As suggested by the previous code reviewer, I kindly suggest that you add icons for language, framework, and skills.
image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Abrar052,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
image
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
> 
> ## [Optional] suggestions:

>- [ ] kindly remove your `node_modules` folder from your `project-feature` GitHub branch as you may have mistakenly pushed  `node_modules` to your branch repo.
node_modules/ has a large size and is commonly excluded from the remote repository. :Smile:

> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 5 above)
Comment (Abrar052) [reply index.html:43]: @topeogunleye can you please tell me which div tag. The tag with class=container-2 or tag with grid-container.

Comment (topeogunleye) [reply index.html:43]: @Abrar052, Kindly use `<section class="grid-container"></section>` instead

**Pull request closed without merging**